config/README.md

```diff
@@ -125,7 +125,32 @@ _your own configuration lines_
 
 ### `.github/workflows/meta.yml`
 
-_At this time there are no configuration options_.
+Add the `[github]` TOML table in `.meta.toml`,
+and set the enabled jobs with the `jobs` key.
+
+```toml
+[github]
+jobs = [
+    "qa",
+    "test",
+    "coverage",
+    "dependencies",
+    "release-ready",
+    "circular",
+    ]
+"""
+```
+
+Extend github workflow configuration with additional jobs
+by setting the values for the `additional_jobs` key.
+
+```toml
+[github]
+additional_jobs = """
+  another:
+    uses: org/repo/.github/workflows/file.yml@main
+"""
+```
```

Comment:
Would you mind adding another documentation section here for the `branch` variable?

I would like to eventually start using tags for this `branch` variable, so we can _safely_ evolve the workflows while at the same time ensuring we don't break existing workflows on repositories 🍀 

Maybe we can rename it `ref` ? 🤔

Reply:
Done